src/styles/variables/_z-index-variables.scss

```diff
@@ -2,13 +2,14 @@
 ========================== */
 $navbar: 400; // should always stay on top, only adjust if you know what you're doing
 $map-overlay: 100;
-$map-attribute-filter: 2;
-$basemap-selector: 2;
-$active-map-item-connection-line: 1;
-$feature-info-content-table-header: 1;
-$search-window-results: 3;
-$cdk-drag-handle: 1;
-$collapsing-flexbox-list-item-hover: 1;
-$data-download-order-queue: 1;
-$scrollbar-width-calculation-container: 0;
-$map-container: -1; // should always stay at the bottom (lower than 0), only adjust if you know what you're doing
+$search-window-results: 4;
+$map-attribute-filter: 3;
+$basemap-selector: 3;
+$active-map-item-connection-line: 2;
+$feature-info-content-table-header: 2;
+$cdk-drag-handle: 2;
+$collapsing-flexbox-list-item-hover: 2;
+$data-download-order-queue: 2;
+$scrollbar-width-calculation-container: 1;
+$map-ui-element: 1; // all elements on the map must have this set
```

Comment:
Why? Shouldn't they automatically be on top of the map? 🤔

Reply:
This was the case when we still had the <map-container> as the first element. Now that it is the last one (which i had to do for tab order), it is rendered above all the other ui elements

Reply:
As discussed - it would probably make more sense to either:
* define the `tab` order - that's maybe not easy possible because the Esri element is hidden beneath their code.
* define the `zindex` for all elements except the map container so that new elements automatically have this without adding that manually for each
  * using a css class that works for all children
  * using a container around all map elements

Because currently we're _misusing_ the `zindex` to define the `tab` order. And you have to add it for each new element as well.